gems/sorbet-runtime/test/types/returns.rb

```diff
@@ -41,7 +41,7 @@ def initialize; end
     end
 
     it 'nils the return for void' do
-      assert_equal(T::Private::Types::Void::VOID, TestReturns.voider)
+      assert_same(T::Private::Types::Void::VOID, TestReturns.voider)
```

Comment:
I changed this because I think it's worth asserting the singleton-ness of the result. LMK what you think!